client/my-sites/comments/constants.js

```diff
@@ -0,0 +1,8 @@
+// Maximum number of comments that will be shown per page.
```

Comment:
I considered moving this file to `client/lib/comments`, but not sure if it's worth it to create a new folder there just for this purpose.

Reply:
I'm personally opposed to it. These constants are only relevant to Comment Management, better not clutter `lib` with something that might confuse other devs.